[Kodi] Unable to resolve root: missing requirement

OH 2.3.0 snapshot 1234
Kodi 2.3.0.201803212307

Removing Kodi from my /conf/services/addons.cfg will let OH start cleanly, but adding it back results in the error. This did not occur in 1232. This is also being reported in the community... https://community.openhab.org/t/after-oh2-3-upgrade-fail-to-install-any-features-error-core-karaf-internal-featureinstaller-failed-installing-openhab/41209/20?u=5iver.

`2018-03-22 17:33:10.744 [ERROR] [org.openhab.core.karaf.internal.FeatureInstaller  ] - Failed installing 'openhab-binding-kodi': Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=openhab-binding-kodi; type=karaf.feature; version="[2.3.0.SNAPSHOT,2.3.0.SNAPSHOT]"; filter:="(&(osgi.identity=openhab-binding-kodi)(type=karaf.feature)(version>=2.3.0.SNAPSHOT)(version<=2.3.0.SNAPSHOT))" [caused by: Unable to resolve openhab-binding-kodi/2.3.0.SNAPSHOT: missing requirement [openhab-binding-kodi/2.3.0.SNAPSHOT] osgi.identity; osgi.identity=org.openhab.binding.kodi; type=osgi.bundle; version="[2.3.0.201803212307,2.3.0.201803212307]"; resolution:=mandatory [caused by: Unable to resolve org.openhab.binding.kodi/2.3.0.201803212307: missing requirement [org.openhab.binding.kodi/2.3.0.201803212307] osgi.wiring.package; filter:="(osgi.wiring.package=org.eclipse.smarthome.config.discovery.upnp)"]]`

--- a/features/openhab-addons/src/main/feature/feature.xml
+++ b/features/openhab-addons/src/main/feature/feature.xml
@@ -190,6 +190,7 @@
 
     <feature name="openhab-binding-kodi" description="Kodi Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
+        <feature>openhab-transport-upnp</feature>
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.kodi/${project.version}</bundle>
     </feature>
 

